docs/source/federation/migrating-from-stitching.md

```diff
@@ -76,7 +84,7 @@ If you're using one of these packages, ensure that after configuring it, your ex
 
 ## Step 2: Register your schemas with a GraphQL registry
 
-We strongly recommend that you register all of your GraphQL schemas with an [external registry](https://principledgraphql.com/integrity#3-track-the-schema-in-a-registry). Doing so improves the reliability of your data graph and maintains a single source of truth to simplify collaboration.
+We strongly recommend that you register all of your GraphQL schemas with an [external registry](https://principledgraphql.com/integrity#3-track-the-schema-in-a-registry). This registry is going to be used to support running the gateway with the supporting service's partial schemas as configuration. Additionally, it enables tracking changes at the service level and protecting the graph from changes that break composition.
```

Comment:
"The registry supports running"

"with the implementing services' partial schemas"